<p>
    Setting an empty value to a key is often useless. But if it is really intended, a comment should be added to explain
    the purpose of the empty property.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
key1=
</pre>
